### PR TITLE
Add support for Webpack style module ids in LESS/CSS

### PR DESCRIPTION
**Pull request by chuckdumont**
As documented here https://github.com/webpack/less-loader#imports

### CODE REVIEW

**Comment by chuckdumont** [comment]
@ddumont, please review

File: jaggr-core/src/main/java/com/ibm/jaggr/core/util/rhino/ReadFileExtFunction.java [hunk 1]
```diff
@@ -62,6 +62,8 @@
 	static final public String FUNCTION_NAME = "readFileExt"; //$NON-NLS-1$
 
 	static final private Pattern NON_AMD_PATH_PAT = Pattern.compile("^(?:[a-z-]+:|\\/|\\.)"); //$NON-NLS-1$
+	static final protected Pattern WEBPACK_MODULE_PAT = Pattern.compile("^~[^/]"); //$NON-NLS-1$
```
Comment (ddumontatibm) [inline jaggr-core/src/main/java/com/ibm/jaggr/core/util/rhino/ReadFileExtFunction.java]: Can we have this in only 1 place?  There are 2 compiled patterns at the moment.

**Comment by ddumontatibm** [comment]
otherwise lgtm

**Comment by chuckdumont** [comment]
Requested change made.

**Pull request merged**